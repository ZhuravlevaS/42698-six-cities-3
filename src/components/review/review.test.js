import React from 'react';
import Review from './review.jsx';
import renderer from 'react-test-renderer';

const review = {
  text: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.`,
  username: `Max`,
  rate: 3,
  date: `December 24, 2018`,
  id: 1
};

it(`<Review/> render`, () => {
  const tree = renderer
    .create(<Review key={review.id} review={review} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
