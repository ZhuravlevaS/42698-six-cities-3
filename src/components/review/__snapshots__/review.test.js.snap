// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Review/> render 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="https://htmlacademy-react-3.appspot.com/six-cities/static/avatar/4.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      karpenter.xo
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "100%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      A quiet cozy and picturesque that hides behind a a river by the unique lightness ofA quiet cozy and picturesque that hides behind a a river by the unique lightness of
    </p>
    <time
      className="reviews__time"
      dateTime="2020-04-01T15:14:31.636Z"
    >
      April 2020
    </time>
  </div>
</li>
`;
